packages/tizen_bundle/example/pubspec.yaml

```diff
@@ -0,0 +1,87 @@
+name: tizen_bundle_example
+description: Demonstrates how to use the tizen_bundle plugin.
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `flutter pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
```

Comment:
Please remove all the comments and any unused dependency (e.g. `cupertino_icons`, `tizen_interop`) in this file. Please refer to other plugins' `example/pubspec.yaml` for how to format this file.

Reply:
Done.